### PR TITLE
Renderyml

### PR DESCRIPTION
**Pull request by eleanorwilliams**
more renderdef.ymls to add. Testing on test32

### CODE REVIEW

**Comment by joshmoore** [comment]
Checked & done:

 * [x] sysgro channel names

File: idr0020-barr-chtog/screenA/idr0020-screenA-renderdef.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# channel names and colours changed from original imported
+
+channels:
+  1:
+    label: "Hoescht"
```
Comment (joshmoore) [inline idr0020-barr-chtog/screenA/idr0020-screenA-renderdef.yml]: Hoechst

File: idr0018-neff-histopathology/experimentA/idr0018-experimentA-renderdef.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# channel names are changed from original imported
+
+channels:
+  1:
+    label: "red"
```
Comment (joshmoore) [inline idr0018-neff-histopathology/experimentA/idr0018-experimentA-renderdef.yml]: Note: this doesn't hold for the 230 grayscale overview slides.

**Comment by eleanorwilliams** [comment]
--exclude

**Comment by sbesson** [comment]
This is now conflicting. Can we close? Are the todos captured somewhere?

**Comment by joshmoore** [comment]
@eleanorwilliams : thoughts on how valuable any of this is? Is it anywhere else?

**Comment by joshmoore** [comment]
thanks, @eleanorwilliams. Moved the card to idr backlog.

**Pull request closed without merging**